contrib/devtools/security-check.py

```diff
@@ -184,6 +198,14 @@ def check_PE_NX(executable) -> bool:
     bits = get_PE_dll_characteristics(executable)
     return (bits & IMAGE_DLL_CHARACTERISTICS_NX_COMPAT) == IMAGE_DLL_CHARACTERISTICS_NX_COMPAT
 
+def check_PE_control_flow_instrumentation(executable) -> bool:
+    stdout = run_command([OBJDUMP_CMD, '-d', '--section=.text', executable])
```

Comment:
Does PE have a `.text` section?

Reply:
Yes, see https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#special-sections, and filltering by --section here prevents additional output from `objdump`.